### PR TITLE
docs: on Python update

### PR DESCRIPTION
**Pull request by discord9**
Update desciption in #279 and #273

### CODE REVIEW

File: docs/user-guide/python-coprocessor/overview.md [hunk 1]
```diff
@@ -16,27 +16,9 @@ We think the python coprocessor in GreptimeDB is a perfect replacement for store
 # Note:
 The Python coprocessor is currently in its experimental phase, and the API may undergo some changes.
 
-Using it also involves some complications. You must set up the correct Python shared library, which can be a bit challenging. In general, you just need to install the `python-dev` package(on most Debian-based system). However, if you are using Homebrew to install Python on macOS, you must create a proper soft link to `Library/Frameworks/Python.framework`.
+Using coprocessor with CPython also involves some complications. You must set up the correct Python shared library, which can be a bit challenging. But you can also use RustPython which doesn't require additional setup.
```
Comment (xtang) [inline docs/user-guide/python-coprocessor/overview.md:19]:
```suggestion
Using coprocessor with CPython also involves some complications. You must set up the correct Python shared library, which can be a bit challenging. But you can also use RustPython which doesn't require additional setup.

```

File: docs/user-guide/python-coprocessor/overview.md [hunk 2]
```diff
@@ -16,27 +16,9 @@ We think the python coprocessor in GreptimeDB is a perfect replacement for store
...
+The recommended way is to use `conda` for managing your Python environment, you must create a Python environment with the same version of Python as the binary you download requires(Or just use docker container) and run the `greptime` binary inside it. A less recommended way is to install the exact version of python manually, and set the `LD_LIBRARY_PATH` environment variable to the directory where the `libpython<VERSION>.so`(the number of <VERSION> depends on the version of Python where ) file is located.
```
Comment (xtang) [inline docs/user-guide/python-coprocessor/overview.md:20]:
```suggestion
The recommended way is to utilize `conda` for managing your Python environment. Firstly, create a Python environment with the same version of Python demanded by the binary you download. Alternatively, you can employ a docker container and execute the `greptime` binary within it.

A less recommended way is to manually install the exact version of Python required and set the `LD_LIBRARY_PATH` environment variable to the directory containing the `libpython<VERSION>.so` file. The version number of `<VERSION>` varies according to the version of Python being used.
```

File: docs/user-guide/python-coprocessor/overview.md [hunk 3]
```diff
@@ -16,27 +16,9 @@ We think the python coprocessor in GreptimeDB is a perfect replacement for store
...
-lib_name=python3.9
-lib_dir=<Your LIB_DIR get from step 1>
-executable=<You python executable path, could come from `which python`>
-pointer_width=64
-build_flags=
-suppress_build_script_link_lines=false
-```
-You might also need to change the `version` and `lib_name` to correspond with your specific version of Python.
\ No newline at end of file
+There is two Backend for Python Coprocessor:
```
Comment (xtang) [inline docs/user-guide/python-coprocessor/overview.md:22]:
```suggestion
There is two Backend for Python Coprocessor:

```

File: docs/user-guide/python-coprocessor/overview.md [hunk 4]
```diff
@@ -16,27 +16,9 @@ We think the python coprocessor in GreptimeDB is a perfect replacement for store
...
+1. RustPython Interpreter: this is supported without need to install any python library, but it is not as fast as CPython Backend. Release Binary without `pyo3` in its name is using RustPython Interpreter. You can still use Python Syntax in RustPython Interpreter, but you can't use any third-parties libs.
```
Comment (xtang) [inline docs/user-guide/python-coprocessor/overview.md:23]:
```suggestion
1. RustPython Interpreter: this is supported without installing any Python library, but it is not as fast as CPython Backend. The Release Binary without `pyo3` in its name uses RustPython Interpreter. While you can still use Python Syntax in RustPython Interpreter,  you can't use any third-parties libs.
```

File: docs/user-guide/python-coprocessor/overview.md [hunk 5]
```diff
@@ -16,27 +16,9 @@ We think the python coprocessor in GreptimeDB is a perfect replacement for store
...
+2. CPython Interpreter: this is the usual and most common python, using it allows you to use all sort of third-parties libs, but you need to install the correct Python shared library. Release Binary with `pyo3` in its name is using CPython Interpreter.
```
Comment (xtang) [inline docs/user-guide/python-coprocessor/overview.md:24]:
```suggestion
2. CPython Interpreter: this is the most commonly used version of Python. It allows you to use all sorts of third-parties libs, but you need to install the correct Python shared library. Any Release Binary with `pyo3` in its name uses CPython Interpreter.
```

**Comment by xtang** [comment]
@discord9 Review done

**Comment by discord9** [comment]
> 

I rebased to #281 including review changes, will close this PR now

**Pull request closed without merging**